src/components/project/project.rules.ts

```diff
@@ -281,26 +301,46 @@ export class ProjectRules {
             Role.FieldOperationsDirector,
           ],
           transitions: [
-            {
-              to: ProjectStep.PendingFinanceConfirmation,
-              type: TransitionType.Approve,
-              label: 'Approve Project',
-            },
-            {
-              to: ProjectStep.PendingZoneDirectorApproval,
-              type: TransitionType.Approve,
-              label: 'Approve for Zonal Director Review',
-            },
-            {
-              to: ProjectStep.FinalizingProposal,
-              type: TransitionType.Reject,
-              label: 'Send Back for Corrections',
-            },
-            {
-              to: ProjectStep.Rejected,
-              type: TransitionType.Reject,
-              label: 'Reject',
-            },
+            ...(isMultiplication
```

Comment:
@CarsonF - Now that this is coded, it makes me wonder if it would be better to have two separate implementations, one for the momentum/intern flow and one for the multiplication flow, instead of splitting logic like this.  I know that wasn't part of what Andre was supposed to do; I was just thinking about it.

Reply:
File's a mess anyways. It's on the chopping block.

In this case theres no need for the outer array, it could be removed.